### PR TITLE
Ruff PYI024

### PR DESCRIPTION
**Pull request by nstarman**
Followup to https://github.com/astropy/astropy/pull/15276

### CODE REVIEW

File: astropy/modeling/bounding_box.py [hunk 1]
```diff
@@ -20,7 +19,9 @@
 __all__ = ["ModelBoundingBox", "CompoundBoundingBox"]
 
 
-_BaseInterval = namedtuple("_BaseInterval", "lower upper")
+class _BaseInterval(NamedTuple):
```
Comment (nstarman) [inline astropy/modeling/bounding_box.py:22]: Does ``_BaseInterval`` need to exist?

**Review by mhvk — APPROVED** [review]
Seeing it again, this is clearly an improvement, so :+1:! And no comment, so am approving. But will not yet merge given the question about modeling.

File: astropy/modeling/bounding_box.py (hunk 1 above)
Comment (WilliamJamieson) [reply astropy/modeling/bounding_box.py:22]: No, but a change like that should be done separately.

**Review by WilliamJamieson — APPROVED** [review]

**Comment by pllim** [comment]
Thanks, all!

**Pull request merged**